src/pages/details/index.tsx

```diff
@@ -46,8 +46,18 @@ export default function Details() {
     // });
 
     return list.sort((a, b) => {
-      const aVote = a.vote === 'yes' ? votes[a.accountId] || '0' : a.totalStakedBalance;
-      const bVote = b.vote === 'yes' ? votes[b.accountId] || '0' : b.totalStakedBalance;
+      const aVote =
+        a.vote === 'yes'
```

Comment:
For either `yes` or `no` vote, we can get the voted stake amount from `votes[accountId][1]`  now, since contract tracks stake amount for both `yes` and `no`.